### PR TITLE
refactor(item): update color and font size

### PR DESCRIPTION
**Pull request by thetaPC**
Issue number: internal

---------

## What is the current behavior?

Playwright was not importing the correct ionic bundle file, which has caused some snapshots to not be correct. This is apparent when a test uses `setContent`. There are several components that are needing their snapshots to be re-generated. In order to not have a PR with an immense amount of snapshots, several PRs will be created to split off the work to make it easier on reviewers.

The snapshots for item are not accurate. This can be verified by comparing it to the test pages, which do import the right bundle file.

## What is the new behavior?

- Updated snapshots
- Updated color and font size to match design

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

[Base branch](https://github.com/ionic-team/ionic-framework/pull/29898)

### CODE REVIEW

File: core/src/components/item/item.ionic.scss [hunk 1]
```diff
@@ -21,6 +22,7 @@
   --padding-start: #{globals.$ionic-space-400};
 
   font-family: globals.$ionic-font-family;
+  font-size: globals.$ionic-font-size-400;
```
Comment (brandyscarney) [inline core/src/components/item/item.ionic.scss:25]: Should this use `body/lg/medium` instead?

File: core/src/components/item/test/lines/item.e2e.ts-snapshots/item-lines-none-icon-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/item/test/lines/item.e2e.ts-snapshots/item-lines-none-icon-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: The label is shifting here but that might be fixed by changing it to `body/lg/medium`

File: core/src/components/item/item.ionic.scss (hunk 1 above)
Comment (thetaPC) [reply core/src/components/item/item.ionic.scss]: https://github.com/ionic-team/ionic-framework/pull/29905/commits/1c66e25c1790d41124cb094c4cbbf3529012af0e

File: core/src/components/item/test/lines/item.e2e.ts-snapshots/item-lines-none-icon-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (thetaPC) [reply core/src/components/item/test/lines/item.e2e.ts-snapshots/item-lines-none-icon-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Let me know if it's not fixed, I'm having a hard time telling the shift: https://github.com/ionic-team/ionic-framework/pull/29905/commits/1c66e25c1790d41124cb094c4cbbf3529012af0e

Comment (brandyscarney) [reply core/src/components/item/test/lines/item.e2e.ts-snapshots/item-lines-none-icon-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: It's still shifting but we can probably look into it later:

https://github.com/user-attachments/assets/72f0e929-dc48-4b6d-8ca1-9bb6be94856b

**Review by brandyscarney — APPROVED** [review]

**Pull request merged**